database/DevHome.Database/Services/RepositoryManagementDataAccessService.cs

```diff
@@ -52,6 +56,7 @@ public RepositoryManagementDataAccessService(DevHomeDatabaseContextFactory datab
             RepositoryName = repositoryName,
             RepositoryClonePath = cloneLocation,
             RepositoryUri = repositoryUri.ToString(),
```

Comment:
`ToString()` is redundant since this type was changed from `Uri` to `string`.

Reply:
Good catch.